### PR TITLE
Alert button

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

File: CSS/alert.css [hunk 1]
```diff
@@ -1,3 +1,52 @@
...
 @import url("base/color.css");
-@import url("base/nav.css");
\ No newline at end of file
+@import url("base/nav.css");
+.alert-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    /* background-color: var(--background-color); */
```
Comment (Namrata-J) [inline CSS/alert.css]: remove this commented properties if possible!!

File: CSS/base/reset.css [hunk 2]
```diff
@@ -169,13 +177,45 @@ li {
...
     height: 35vh;
     border: none;
     overflow-x: hidden;
+    margin: 0rem 0.5rem 1rem 0.4rem;
 }
 
 .iframe-child_class {
     overflow: auto;
-    margin: 0;
+    /* margin: 0rem; */
```
Comment (Namrata-J) [inline CSS/base/reset.css]: here as well

File: CSS/alert.css (hunk 1 above)
Comment (ajit-sharmaGH) [reply CSS/alert.css]: Okk will do that

**Comment by JantuDeb** [comment]
> _No description provided._

Please add PR description

File: CSS/alert.css [hunk 3]
```diff
@@ -1,3 +1,52 @@
...
+    border: 1px solid var(--dark-primary);
+    padding: 2rem 10% 2rem 10%;
+    border-radius: 1rem;
+}
+
+.alert-btn-all {
+    margin: 0.3rem;
+    border: 2px solid var(--primary);
+    font-weight: bold;
+}
```
Comment (JantuDeb) [inline CSS/alert.css]: `.alert-btn-all` can be reduced to `.alert-btn`

**Review by JantuDeb — CHANGES_REQUESTED** [review]
Suggested some changes

**Review by Raghav888 — APPROVED** [review]
Approved

**Pull request merged**